config/role/role.yaml

```diff
@@ -43,8 +48,11 @@ role:
     - "hs:hs_auth:api:v2:GetUsers"
     - "hs:hs_auth:api:v2:GetTeams"
     - "hs:hs_auth:api:v2:GetAuthorizedResources"
-    - "hs:hs_apply:apply"
+    - "hs:hs_apply:Dashboard:dashboard"
+    - "hs:hs_apply:Application:apply"
+    - "hs:hs_apply:Application:updatePartialApplication"
+    - "hs:hs_apply:Application:submitApplication"
+    - "hs:hs_apply:Application:cancel"
```

Comment:
Do volunteers need to be able to create applications?

Reply:
Good point, probably not